salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -141,7 +141,7 @@
                 var filter = ' p em strong strike u sub sup; a[!href](salsah-link)';
 
 				var config = {
-					language: 'de', // customize language
+					language: SALSAH.userprofile.userData.lang === undefined ? 'de' : SALSAH.userprofile.userData.lang,
```

Comment:
by the way, should the default language remain German? 
(by default the rest of the page is in English)